config/registry/default_schema.yaml

```diff
@@ -195,14 +181,76 @@ dataUnits:
             Timestamp of the beginning of the Visit.  This should be the
             same as the datetime_begin of the first Exposure associated
             with this Visit.
+        -
+          name: datetime_end
+          type: datetime
+          doc: >
+            Timestamp of the end of the Visit.  This should be the
+            same as the datetime_end of the last Exposure associated
+            with this Visit.
```

Comment:
The most important place to enforce it is probably the raw data ingest script.  Since it (and a number of other constraints) can probably be verified via SQL, it might make sense to have a script we can run explicitly to check these things.  I doubt trying to add complex table constraints is worthwhile; I would guess those are pretty RDBMS-specific.